docs/tutorials/galga.md

```diff
@@ -382,7 +382,7 @@ game.onUpdateInterval(1000, function () {
 
 ## Step 8
 
-**How can you win if bogeys [__*spawn*__](#spawnd "appear suddenly") right on top of you?**
+**How can you win if bogeys** [__*spawn*__] (#spawnd "appear suddenly") **right on top of you?**
```

Comment:
```suggestion
**How can you win if bogeys** [__*spawn*__](#spawnd "appear suddenly") **right on top of you?**
```

I don't think there should be a space here (can't remember if it would actually break parsing or if md allows spaces in links for sure)